roles/ipasmartcard_client/library/ipasmartcard_client_get_vars.py

```diff
@@ -64,7 +64,12 @@
 
 import sys
 from ansible.module_utils.basic import AnsibleModule
-from ipaplatform.paths import paths
+MODULE_IMPORT_ERROR = None
```

Comment:
I'd prefer that this variable is set after the fact, in the `else` clause for the `try`.

Reply:
Done